README.md

```diff
@@ -88,6 +88,8 @@ All results are private by default. Use this option to upload reports to LHCI's
 temporaryPublicStorage: true
 ```
 
+[⚙️ See an example workflow with this option](https://github.com/treosh/lighthouse-ci-action/actions?workflow=LHCI-temporary-storage)
```

Comment:
nit: these seem redundant, but could provide context, so I could take them or leave them :man_shrugging:

Reply:
agreed, I thought it would be useful for a context, but it just adds a clutter to the docs.